src/components/menubar/menubar.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import IconButton from '@material-ui/core/IconButton';
+import Helmet from 'react-helmet';
+import PropTypes from 'prop-types';
+
+import { withStyles } from '@material-ui/core/styles';
+
+import acmhack from './acmhack.svg';
+import hackLogo from './hack-logo.png';
+import menubtn from './menu.svg';
+import ButtonBar from './buttonbar';
+import { Collapse } from '@material-ui/core';
+
+const StyleHeader = () =>
+	<Helmet>
+		<link href="https://fonts.googleapis.com/css?family=Poppins:400,500" rel="stylesheet"/>
+	</Helmet>;
+
+const styles = theme => ({
+	logohome: {
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+		cursor: 'pointer'
+	},
+	toolbar: {
+		backgroundColor: 'white',
+		color: '#352A3A',
+		justifyContent: 'space-between',
+		padding: '0% 6%'
+	},
+	logo: {
+		height: theme.typography.fontSize * 1.8,
+		margin: theme.spacing.unit
+	},
+	clubname: {
+		height: theme.typography.fontSize * 1.2
+	},
+	menubtn: {
+		// desktop
+		display: 'none',
+		[theme.breakpoints.down('xs')]: {
+			// mobile
+			display: 'inline'
+		}
+	},
+	desktopMenuBar: {
+		// desktop
+		display: 'inline',
+		[theme.breakpoints.down('xs')]: {
+			// mobile
+			display: 'none'
+		}
+	},
+	mobileMenuBar: {
+		// desktop
+		position: 'fixed',
+		zIndex: '1',
+		display: 'none',
+		overflow: 'hidden',
+		backgroundColor: '#f5f5f5',
+		width: '100%',
+		/* eslint-disable-next-line max-len */
+		boxShadow: '0px 2px 4px -1px rgba(0,0,0,0.2), 0px 4px 5px 0px rgba(0,0,0,0.14), 0px 1px 10px 0px rgba(0,0,0,0.12)',
+		[theme.breakpoints.down('xs')]: {
+			// mobile
+			display: 'inline'
+		}
+	},
+	mobileBtnContainer: {
+		display: 'flex',
+		flexDirection: 'column',
+		padding: '3%'
+	}
+});
+
+class MenuBar extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			menuOpen: false
+		};
+		this.toggleMenu = this.toggleMenu.bind(this);
+	}
+
+	toggleMenu() {
+		const { menuOpen } = this.state;
+		this.setState({
+			menuOpen: !menuOpen
+		});
+	}
+
+	render() {
+		const { classes } = this.props;
+		const { menuOpen } = this.state;
+
+		return (
+			<React.Fragment>
+				<StyleHeader/>
+				<AppBar position="sticky">
+					<Toolbar className={classes.toolbar}>
+						<div className={classes.logohome}>
+							<img src={hackLogo} className={classes.logo} />
+							<img src={acmhack} className={classes.clubname} />
+						</div>
+						{/* Desktop menu Bar */}
+						<div className={classes.desktopMenuBar}>
+							<ButtonBar />
+						</div>
+						{/* This button only shows on mobile */}
+						<IconButton onClick={this.toggleMenu} className={classes.menubtn}>
+							<img src={menubtn} />
```

Comment:
How about we use Material UI's `menu` SVG icon rather than rolling our own `<img>` and SVG? See https://material-ui.com/style/icons/#svg-material-icons.